core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.spec.ts

```diff
@@ -183,15 +193,15 @@ describe('DotFormComponent', () => {
     beforeEach(() => {
         spectator = createComponent({
             props: {
-                formData: LAYOUT_MOCK
+                formData: CONTENT_FORM_DATA_MOCK
             }
         });
     });
 
     describe('initilizeForm', () => {
         it('should initialize the form group with form controls for each field in the `formData` array', () => {
             const component = spectator.component;
-            component.formData = LAYOUT_MOCK;
+            component.formData = CONTENT_FORM_DATA_MOCK;
```

Comment:
You can use `spectator.setInput('formData', CONTENT_FORM_DATA_MOCK);` to simulate the input and trigger the change detection automatically. With that can delete the 203 and 205.